src/main/java/com/example/fizzbuzz/controller/FizzBuzzDBController.java

```diff
@@ -0,0 +1,67 @@
+package com.example.fizzbuzz.controller;
+
+import com.example.fizzbuzz.model.ConnectFizzBuzzDB;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+import java.sql.Connection;
+import java.sql.Statement;
+
+@RestController
+public class FizzBuzzDBController {
+
+    public static String setResult(Integer myNumber){
+        String result = "";
+        if (myNumber % 3 == 0){
+            result += "Fizz";
+        }
+        if (myNumber % 5 == 0){
+            result += "Buzz";
+        }
+        return result;
+    }
+
+
+    public static void connectionAndExecute(String query){
+        Connection connection = null;
+        Statement statement = null;
+
+        ConnectFizzBuzzDB objConnectFizzBuzzDB= new ConnectFizzBuzzDB();
+        connection = objConnectFizzBuzzDB.get_connection();
+
+        try {
+            statement = connection.createStatement();
+            statement.executeUpdate(query);
+        } catch (Exception e) {
+            e.printStackTrace();
+        }
+    }
```

Comment:
I think these 2 methods contain logic which is not related to mapping itself, what do you think about moving these 2 methods to some new services?
For example `FizzBuzzResultService` with setResult method and `FizzBuzzDBService` with connectionAndExecute method